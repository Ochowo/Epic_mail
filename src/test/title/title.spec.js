import React from 'react';
import { shallow } from 'enzyme';
import Title from '../../components/title/title';

const setUp = (props = {}) => {
  const component = shallow(<Title {...props} title="create an account" />);
  return component;
};
describe('Title component', () => {
  let component;
  beforeEach(() => {
    component = setUp();
  });
  it('should render without errors', () => {
    expect(component).toMatchSnapshot();
  });
  it('should render children', () => {
    const wrapper = component.find('h2');
    expect(wrapper.find('.heading').exists()).toBe(true);
  });
});
