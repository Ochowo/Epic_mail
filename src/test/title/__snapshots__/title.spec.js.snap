// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component should render without errors 1`] = `ShallowWrapper {}`;
